tokio/src/time/driver/tests/test_inner.rs

```diff
@@ -0,0 +1,57 @@
+#![warn(rust_2018_idioms)]
```

Comment:
nit: i'm pretty sure this is unnecessary, since we enable it globally for the whole crate here:
https://github.com/tokio-rs/tokio/blob/4a24c7063b80f31106244b955942a5162a426cd1/tokio/src/lib.rs#L10

in the integration tests, we add these attributes because they're considered separate crates, and thus the attributes in `lib.rs` don't apply; but unit tests like this one are part of the same crate and should have all the attributes we set globally.

Reply:
Heard, good catch. Will clean up.

Reply:
Resolved in 5e1fd00.